shlr/qnx/src/utils.c

```diff
@@ -182,8 +185,7 @@ extract_unsigned_integer (const uint8_t *addr, int len, int be) {
 	return retval;
 }
 
-int
-i386nto_regset_id (int regno) {
+int i386nto_regset_id (int regno) {
```

Comment:
these are kinda big and were pulled from `gdb`, so i'd prefer not to clutter the `core.c` file with it, additionally there soon may appear similar functions for ARM32 arch